README.md

```diff
@@ -24,14 +24,36 @@ To get started with Finch on macOS, the prerequisites are:
 * Intel or Apple Silicon M1 system for macOS
 * Recommended minimum configuration is 2 CPU, 4 GB memory
 
+##### From Go
+
+The easiest way to install the Finch is to use Go way by issuing the following command:
+
+```sh
+$ go install github.com/runfinch/finch/cmd@v0.1.0
+```
+
+##### From Releases
+
 Download a release package for your architecture from the [project's GitHub releases](https://github.com/runfinch/finch/releases) page, and once downloaded double click and follow the directions.
 
+##### From Source
+
+If you want to build and compile the source code, follow the instructions below:
+
+```sh
+$ git clone git@github.com:runfinch/finch.git
+$ cd finch
+$ make install
+```
+
+#### Running the VM
+
```

Comment:
I'm a bit hesitant to add this because probably only `finch version` would work as the underlying VM is not there, and README is for users (say, compared to `CONTRIBUTING.md`, which is for devs), so instructions given here may need to yield a fully-functional thing. WDYT? /cc @ningziwen

Reply:
I'm not sure if I clearly follow your concern. All instructions here are fully-functional. For both installation methods, `finch version` works as expected.

Reply:
My bad, I meant that `finch version` would probably be the **only** command that works. Important commands like VM and container ones won't work:

```sh
➜  finch git:(main) ./_output/bin/finch pull alpine
FATA[0000] fork/exec /local/home/davidhyc/davidhsingyuchen/finch/_output/lima/bin/limactl: no such file or directory
➜  finch git:(main) ./_output/bin/finch vm init
FATA[0000] fork/exec /local/home/davidhyc/davidhsingyuchen/finch/_output/lima/bin/limactl: no such file or directory
```

Reply:
Ohhh, see now. Let me check what can I do to fix this. Maybe we should also install `limactl` if does not exist. Wdyt?

Reply:
Lima and its deep dependencies are added by lima-and-qemu target in [makefile](https://github.com/runfinch/finch/blob/main/Makefile#L84), which is downloaded from the binaries generated by finch-core.

Basically finch go package is just the top layer of the whole Finch application. So it is expected that it can't work independently without the lower layers. Let me know if you have different thoughts.

Reply:
I think I got lost on this and was thinking on the providing the best UX as possible.

For example, limactl, automatically [downloads](https://github.com/lima-vm/lima/blob/79a8791f965515e21f01f84e1230e05c9272fb98/pkg/downloader/downloader.go) all necessary artifacts (binaries etc.). And I have been thinking to implement the similar use case which would require the complete removal of the `lima-and-qemu` target in the makefile. So that we let finch do this job in the background. Eventually we will achieve,

* significantly reduce the binary size of release (don't need to maintain external artifacts)
* should work as a standalone binary
* compatible with `go install`, `brew`, and other package managers

Reply:
It strikes me that the current install workflow makes total sense for those who just want to _use_ Finch, as it resolves all the dependencies in one step without giving much room for error.

Using Go or installing from source would probably confuse some users along the way as they could make a mistake or miss a step easily. Not to say that someone contributing or hacking Finch wouldn't use `go` or install from source but we probably want to keep these separate workflows and off the README. 

@Dentrax what do you think?